account_invoice_fiscal_position_update/models/account_move.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2011-2020 Julius Network Solutions SARL <contact@julius.fr>
+# Copyright 2014-2020 Akretion France (http://www.akretion.com)
+# @author Mathieu Vatel <mathieu _at_ julius.fr>
+# @author Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    @api.onchange("partner_id")
+    def _onchange_partner_id(self):
+        fiscal_position = self.fiscal_position_id
+        res = super()._onchange_partner_id()
+        if fiscal_position != self.fiscal_position_id:
+            self._onchange_fiscal_position_id()
+        return res
+
+    @api.onchange("fiscal_position_id")
+    def _onchange_fiscal_position_id(self):
```

Comment:
Don't call it as the standard name, or it can conflict with other module adding another onchange for the same field. The guideline is to put the module name in the method:

```suggestion
    def _onchange_fiscal_position_id_account_invoice_fiscal_position_invoice(self):
```